### ISSUE TITLE
Remove unpublished fork `ricardoboss/easysvg`

### ISSUE THREAD

**Issue by JellyBellyDev** [badges/poser#87]
### Bug Report

|    Q            |   A
|---------------- | ------
| BC Break        | no
| Library Version | 2.3.0
| PHP version     | 7.4

#### Summary

Remove unpublished fork `ricardoboss/easysvg` deps to composer.json as soon as the PR is released: https://github.com/kartsims/easysvg/pull/32

Assigned to @ricardoboss :)
Ok?

**Comment by antonkomarev** [comment]
Cannot install v2.3

```shell
/app # composer require badges/poser:^2.3
./composer.json has been updated
Running composer update badges/poser
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires badges/poser ^2.3 -> satisfiable by badges/poser[v2.3.0, 2.x-dev].
    - badges/poser[v2.3.0, ..., 2.x-dev] require kartsims/easysvg ^2.0 -> found kartsims/easysvg[dev-master, 1.0] but it does not match the constraint.
```

**Comment by ricardoboss** [comment]
You need to add my fork as a repository so composer can find version 2.3:

```json
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/ricardoboss/easysvg"
        }
    ]
```

Once my fork (https://github.com/kartsims/easysvg/pull/32) is merged, this will no longer be necessary

**Comment by antonkomarev** [comment]
@ricardoboss thank you for the example. How this dependency is better than solution with font size detection implemented in other badges?

**Comment by ricardoboss** [comment]
In my original PR, I used this library to not only calculate the font size, but also to generate SVG paths for the letters in the badge. Later, after revisiting which fonts are used, I merely used the library for it's features with letter spacing and kerning.

If I remember correctly, the current implementation for text width calculation doesn't support this. Also, the `for-the-badge` style needs different styling in that the value needs to be bold.

**Comment by ricardoboss** [comment]
@JellyBellyDev @antonkomarev A new version of `kartsims/easysvg` was published (`2.4`). I will open a PR to close this issue.
